### ISSUE TITLE
installing unetbootin

### ISSUE THREAD

**Issue by Billy25780** [unetbootin/unetbootin#386]
Ran your scripts to install Unetbootin, but it tells me it can't find the package.

wolf@wolf1:~$ sudo apt-get install unetbootin
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package unetbootin
wolf@wolf1:~$ sudo apt install unetbootin
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package unetbootin

**Comment by Billy25780** [comment]
Update: in terminal looked at the download list and it says: 404  Not Found [IP: 185.125.190.80 443]
Maybe your links need updating.

**Comment by Billy25780** [comment]
And it might not exist. I looked through the list of distros and there isn't one for Jammy (Ubuntu 22)

**Comment by Parsec86** [comment]
Agree. No distro for jammy (neither for noble), latest available in _https://ppa.launchpadcontent.net/gezakovacs/ppa/ubuntu/dists/_ is focal (20).
Anyway, changing release from jammy to focal allows flawless installation (in jammy - 22).
Should work in noble (24) too.

**Comment by papifr** [comment]
Hello Parsec86
I have the same problem with Lubuntu Noble
I want to install UNetbootin on my Noble version using Focal as you suggested.
How do I do this ???
